pkg/binutils/latest.go

```diff
@@ -61,6 +61,8 @@ func GetLatestReleaseVersion(releaseURL string) (string, error) {
 // the given repo and version, and installs it into the apps `bin` dir.
 // NOTE: If any of the underlying URLs change (github changes, release file names, etc.) this fails
 // The goal MUST be to have some sort of mature binary management
+//
+// Deprecated: Use GetLatestReleaseVersion
```

Comment:
so we need to find a replacement in E2E probably.
anyway, GetLatestReleaseVersion only get the version string, and this function installs an archive. so not the same.

Reply:
Created a ticket to fix this #316